hack/terraform/aws/variables.tf

```diff
@@ -5,10 +5,18 @@ variable owner {
   default = null
 }
 
-variable aws_vm_type {
+variable aws_vm_type_1 {
   default = "t2.micro"
 }
 
+variable aws_vm_type_2 {
+  default = "t3.medium"
+}
+
+variable "ssh_public_key" {
+  default = "~/.ssh/id_rsa.pub"
```

Comment:
`aws_key_pair_name` is required, so can we reuse that key?

Reply:
aws_key_pair_name - This is used by AWS to deploy the VMs with key. But the init_script requires the public key, which will be copied on to Windows VM (in .ssh/authorized_keys), so that test will be able to ssh into VMs.